packages/opencensus-core/src/exporters/types.ts

```diff
@@ -25,8 +24,13 @@ export interface Exporter extends modelTypes.SpanEventListener {
    * Sends a list of root spans to the service.
    * @param rootSpans A list of root spans to publish.
    */
-
   publish(rootSpans: modelTypes.RootSpan[]): Promise<number|string|void>;
 }
 
+/** Defines a StatsEventListener interface */
```

Comment:
Not sure if this is within the scope of this change, but could you make this more descriptive?

Reply:
This is related to the stats exporter. Changed the comments a little bit to be more clear. Please, tell me if it's still confusing.

Reply:
`stats exporter` is better, as it shows who should be implementing it. To make it slightly more explicit:

```
An interface that describes the possible events that will be emitted from a Stats instance. Stats exporters should implement this interface.
```